executor/show_placement.go

```diff
@@ -127,3 +127,24 @@ func (e *ShowExec) fetchShowPlacementLabels(ctx context.Context) error {
 
 	return nil
 }
+
+func (e *ShowExec) fetchShowPlacement(_ context.Context) error {
+	err := e.fetchAllPlacementPolicies()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
There is no difference with `return e.fetchAllPlacementPolicies()`. Any reason not to inline the function?

Reply:
`SHOW PLACEMENT` show placement settings for policies, schemas, tables and partitions. So I created the function to separate these logics.  But currently only policies are available so they seems the same.

Reply:
Makes sense.